corehq/apps/userreports/views.py

```diff
@@ -438,6 +438,9 @@ def post(self, request, *args, **kwargs):
                 'source_type': app_source.source_type,
                 'source': app_source.source,
             }
+            if 'registry_slug' in app_source._fields:
+                if app_source.registry_slug != '':  # don't add to params unless a registry is selected
```

Comment:
With the changes above `registry_slug` will always be present in the fields.
```suggestion
            if app_source.registry_slug != '':  # don't add to params unless a registry is selected
```